tests/protocol/mockwiredata.py

```diff
@@ -123,6 +123,10 @@
 DATA_FILE_STATUS_BLOB["ext_conf"] = "wire/ext_conf_mock_status_blob.xml"
 
 
+def is_ga_manifest_request(url):
+    return "manifest_of_ga.xml" in url
```

Comment:
I wonder why this is showing up in your PR. Like the merge should've taken care of it. Anyways, approved